pkg/obfuscate/obfuscate.go

```diff
@@ -240,6 +240,12 @@ type JSONConfig struct {
 	ObfuscateSQLValues []string `mapstructure:"obfuscate_sql_values"`
 }
 
+// CommandExecutionConfig holds the configuration settings for command execution obfuscation
+type CommandExecutionConfig struct {
+	// Enabled specifies whether this feature should be enabled.
+	Enabled bool `mapstructure:"enabled"`
```

Comment:
We must add a way to control this through env var, here is an [example](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/setup/apm.go#L24). The new parameter's syntax should be included in the release note.

The defaulting logic for obfuscation config must also be updated to cover the new field ([here](https://github.com/DataDog/datadog-agent/blob/6294adc25e841d566a78bc7fc2db08b2f3491d87/comp/trace/config/setup.go#L400-L475))

Reply:
All should be applied in 3a81979